### ISSUE TITLE
Images not displaying 

### ISSUE THREAD

**Issue by maodoanminnh** [KQMATH/moodle-mod_capquiz#205]
**Describe the bug**
I installed Capquiz on Moodle 3.10 and 3.11, the same problem happened: images in questions are not displayed.
**To Reproduce**
1. Add a question to the question bank, MC or embedded cloze, with images UPLOADED to the question, not with URLs linked to a public source.
2. Preview the questions to make sure everything is good.
3. Create a Capquiz activity.
4. Add the questions to the quiz and publish it and log in as a student to do it.

**Expected behavior**

All images are not loaded. Some have symbols of broken images, others disappearing completely.

**Screenshots**
![image](https://user-images.githubusercontent.com/89524105/139537938-3726e41a-11bb-4aee-b709-f1e6cb43a3de.png)

**Comment by hgeorgsch** [comment]
Thanks a lot for the import @maodoanminnh 
I have reproduced and can confirm the bug.

I tried to import the question both in Quiz and in CAPQuiz.
The question is displayed with different image URLs in the two quizzes, and only the URL used in Quiz works.  The URL used in CAPQuiz gives a 404 (file not found).

I get the same error when the question is created from the CAPQuiz interface and used in CAPQuiz.

**We need to figure out why the URL can change differently in CAPQuiz.**

Note.  Using an image by URL results in a «Cross-Origin Read Blocking» error (consistently).
Note 2. Exporting the question as XML embeds the image file in base64 encoding.  I have not tried to reimport this to see what happens.
